app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def update_comment_counter
+    post.increment(:comment_counter)
+  end
```

Comment:
- [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the ` #update_comment_counter` method every time a new comment has been saved to database, then make the `update_comment_counter` method private can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comment_counter method every time a new post has been saved to database, then make the `update_comment_counter` method private